packages/runtime/src/runtimes/react/controls/style.ts

```diff
@@ -115,8 +118,30 @@ export type StyleControlFormattedValue = string
 export function useFormattedStyle(
   styleControlData: StyleControlData | undefined,
   controlDefinition: StyleControlDefinition,
+  control: StyleControl | null,
 ): StyleControlFormattedValue {
   const style = useStyleControlCssObject(styleControlData, controlDefinition)
+  const elementRef = useRef<Element | null>(null)
+  const guid = useId().replaceAll(':', '')
```

Comment:
How come we had to replace colons here?

Reply:
We're removing the colons because `useId` returns a string wrapped with colons, e.g. ":r0:", and we cannot use a colon in a class name.